ip_localities.py

```diff
@@ -0,0 +1,79 @@
+from arcgis import GIS
+import pandas as pd
+import math
+
+
+def meters_to_long(x):
```

Comment:
I would recommend writing out `longitude` here so it's easier to read later on.

Reply:
Also, give me some docstrings pls : https://www.python.org/dev/peps/pep-0257/
Ex : 
```
""" Converts meters to longitude.
Parameters
------------
x : float? (this could also be a more informative name...)
   The quantity in meters to be converted.

Returns
--------
longitude : float
   The longitude conversion of the given quantity.
"""
```